### ISSUE TITLE
Typing cursor should be in the search field upon loading

### ISSUE THREAD

**Issue by poiNt3D** [nextcloud/notes#1373]
**Is your feature request related to a problem? Please describe.**

I'm always frustrated when i open the notes web page and start typing into the last note, which is already complete

**Describe the solution you'd like**
When you load the page, keyboard cursor should be in the search field.

**Comment by user8446** [comment]
Agreed, the first thing I do is close the navigation and click on search. There is already an [open issue](https://github.com/nextcloud/notes/issues/1079) to remember the last setting for the navigation which would fix that.
